slides.md

```diff
@@ -156,57 +156,97 @@ import anotherFunction, {aFunction} from "./someFile";
 
 ---
 # Hello Todo!
-1. Edit `App.js` so that the list rendered from an array:
+Currently `App.js` is quite boring, it would be great if instead of having the todos hardcoded, we could somehow cycle through an array of todos and show a `Text` per element.
```

Comment:
nit: missing a comma after `Currently`

Grammarly always bugs me about this 😅

Reply:
22c4882